dapps/marketplace/src/components/GrowthCampaignBox.js

```diff
@@ -40,6 +38,16 @@ const renderNotEnrolledBox = openmodalonstart => {
 }
 
 const renderEnrolledBox = (props, walletAddress) => {
+  const campaigns = props.growthCampaigns
+  if (campaigns.length == 0) {
+    return <h5 className="p-2">No campaigns detected</h5>
```

Comment:
Should these be wrapped? 🙂

Reply:
Sorry @sparrowDom I didn't read the "WIP". 😇 I skimmed this to see if it would conflict with something that I was doing.

Reply:
@micahalcorn thanks for this catch. This part has somehow evaded translation in the current master: https://github.com/OriginProtocol/origin/blob/master/dapps/marketplace/src/components/GrowthCampaignBox.js#L59 and would also probably dodge it with this refactoring. Thanks for noticing...